boards/arm/mimxrt1050_evk/mimxrt1050_evk.dts

```diff
@@ -220,7 +220,6 @@ zephyr_udc0: &usb1 {
 
 &usdhc1 {
 	status = "okay";
-	no-1-8-v;
```

Comment:
why remove this?

Reply:
We have been using the `no-1-8-v` flag to block the usdhc driver from switching cards to 1.8V signalling, because 1.8V signalling levels enable higher clock speeds. The USDHC driver we currently have in the tree typically would fail to transfer data from the SD card at these higher clock speeds, so we disabled using 1.8V altogether with this flag. However, the RT1050 does have the required circuitry to switch an SD card to 1.8V signalling levels, so with the new SDHC driver we can reenable this feature.